common-lib/src/main/scala/com/gu/mediaservice/model/usage/PrintUsageMetadata.scala

```diff
@@ -41,13 +41,20 @@ case class PrintUsageMetadata(
   type LongElement = List[(String,Long)]
   type IntElement = List[(String,Int)]
 
+  val calculatedPublicationName =
+    if (publicationName.trim.isEmpty) publicationCode match {
```

Comment:
@akash1810 I think they might sometimes be whitespace at the moment, this is within our control though, @blishen is looking `t it.